pkg/controller/httpserver/runner.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package httpserver
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("httpserver")
+
+// DefaultBindAddress sets the default bind address for the HTTP server listener
+var DefaultBindAddress = ":8080"
```

Comment:
I don't see any reason why this is public

Reply:
if they want to use it as default args value